site/_posts/2017-11-06-call-me-gapri.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Last Week in ManageIQ: Please call me Gapri if it helps you relax"
+author: Roman Blanco
+date: 2017-11-07
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Wohoo! It's here again! All the features we were working on in last truly _Fine_ summer, are now released in new version, **Gaprindashvili**.
```

Comment:
I like this 😄  perhaps it should be slightly reworded to "working on over the truly fine summer" as opposed to "in last"?